src/Microdown/MicEnvironmentBlock.class.st

```diff
@@ -157,17 +157,13 @@ MicEnvironmentBlock >> environmentName [
 
 { #category : #accessing }
 MicEnvironmentBlock >> extractFirstLineFrom: aLine [
-	"we got <! env ...x !> foo bar or <! env ...x and we return env ...x "
+	"we got <! env ...x !> <! env ...x and we return env ...x "
 	
-	| extract |
 	firstLine := super extractFirstLineFrom: aLine.
-	firstLine := firstLine trimBoth.
-	extract := firstLine copyUpToSubstring: EnvironmentClosingBlockMarkup.
-	"we got !> ..."
-	^ extract size < firstLine size 
+	^ (firstLine endsWith: EnvironmentClosingBlockMarkup) 
 		ifTrue: [ 
 			isClosed := true.
-			extract trimRight ]
```

Comment:
This will undo a change done by Stef a few days ago. 
Did you make sure to pull the Microdown/dev changes into your dev first?

Reply:
Yes i double dheck that i was up-to-date on my branch dev